Importer/Interpreter/Exception/FieldValueException.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Netdudes\ImporterBundle\Importer\Interpreter\Exception;
+
+class FieldValueException extends InterpreterException
```

Comment:
Yes, but the idea is to be less specific 😄 . The exception handles only invalid values. Also, "field" is a fairly broad term that does not provide that much info.

It is still under the interpreter namespace, so `InvalidValueException` is a better name thand `FieldValueException`. Unless you have another suggestion?